onnxruntime/contrib_ops/rocm/bert/gemm_fast_gelu.cc

```diff
@@ -54,10 +55,13 @@ Status GemmFastGelu<T>::ComputeInternal(OpKernelContext* ctx) const {
   const HipT alpha = ToHipType<T>::FromFloat(1.0f);
   const HipT beta = ToHipType<T>::FromFloat(0.0f);
 
-  return LaunchGemmFastGeluKernel<HipT>(
+  using onnxruntime::rocm::tunable::blas::BlasOp;
+
+  return blas::row_major::GemmFastGelu(
       IsTunableOpEnabled(),
       Stream(), RocblasHandle(),
-      transa, transb,
+      transa ? BlasOp::Trans : BlasOp::NonTrans,
+      transb ? BlasOp::Trans : BlasOp::NonTrans,
       static_cast<int64_t>(helper.M()), static_cast<int64_t>(helper.N()), static_cast<int64_t>(helper.K()),
```

Comment:
Isn't `static_cast<int64_t>` redudent after the change?

Reply:
It's still need, return type of `helper.M()`  is `ptrdiff_t`.